content/sensu-enterprise/3.8/changelog.md

```diff
@@ -60,6 +61,22 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
 - [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
 - [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)
 
+## Enterprise 3.8.0 Release Notes {#enterprise-v3-8-0}
+
+**April 8, 2020** &mdash; Sensu Enterprise version 3.8.0 has been
+released and is available for immediate download. Please note the
+following improvements:
+
+### CHANGES {#enterprise-v3-8-0-changes}
+
+Built on Sensu Core 1.10.0:
+
+- **NEW**: Updates the gem name from `sensu` to `sensu-eol`, which is built from https://github.com/sensu/sensu-eol.
```

Comment:
```suggestion
```

Reply:
This isn't really a user-facing detail, so we should just describe it above as an internal release.